hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -3023,4 +3023,15 @@
       will create intermediate directories.
     </description>
   </property>
+
+  <property>
+    <name>ozone.client.checksum.combine.mode</name>
+    <value>MD5MD5CRC</value>
+    <tag>OZONE, CLIENT</tag>
+    <description>
+      The combined checksum type [MD5MD5CRC / COMPOSITE_CRC]
+      determines which algorithm would be used to compute checksum for
+      file checksum. Default checksum type is MD5MD5CRC.
```

Comment:
Could you add few more details about the purpose of checksum types. How about something like,

`COMPOSITE_CRC type to calculate combined CRC of the whole file, the lower-level chunk/block checksums are combined into file-level checksum. MD5MD5CRC type calculates MD5 of MD5 of checksums of individual chunks.`

Reply:
OK, its addressed in latest patch.